esmvaltool/cmorizers/data/formatters/datasets/mobo_dic_mpim.py

```diff
@@ -0,0 +1,161 @@
+"""ESMValTool CMORizer for MOBO-DIC_MPIM data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    https://www.ncei.noaa.gov/data/oceans/ncei/ocads/data/0221526/
+
+Last access
+    20221103
+
+Download and processing instructions
+    Download the file MOBO-DIC_MPIM_monthly_clim.nc
+
+"""
+
+import logging
+import warnings
+from datetime import datetime
+from pathlib import Path
+
+import iris
+from cf_units import Unit
+from dask import array as da
+from iris import NameConstraint
+from iris.coords import CellMethod, DimCoord
+
+from esmvaltool.cmorizers.data import utilities as utils
+
+logger = logging.getLogger(__name__)
+
+
+def _callback_fix_missing_value(cube, field, _):
+    """Create masked array from missing_value."""
+    if hasattr(field.cf_data, 'missing_value'):
+        missing_value = float(field.cf_data.missing_value)
+        logger.info("Fixing missing value (%f)", missing_value)
+        cube.data = da.ma.masked_equal(cube.core_data(), missing_value)
+
+
+def _fix_climatological_time(cube):
+    """Fix climatology coordinate."""
+    time_units = Unit('days since 1950-01-01 00:00:00', calendar='standard')
+
+    # Following the doc the covered time period of the climatology is
+    # January 2004 to December 2017 (Use 2011 as the "mean" year). See
+    # https://www.ncei.noaa.gov/access/metadata/landing-page/bin/
+    # iso?id=gov.noaa.nodc%3A0221526
+    time_points = time_units.date2num(
+        [datetime(2011, m, 15) for m in range(1, 13)]
+    )
+    time_bounds = [
+        [datetime(2004, m, 1), datetime(2017, m + 1, 1)] for m in range(1, 12)
```

Comment:
so bounds are [[1-Jan-2004, 1-Feb-2017], [1-Feb-2004, 1-Mar-2017], ..., [1-Nov-2004, 1-Dec-2017]]? Beats me if I understand them :grin:

Reply:
Yeah, that's a climatology - the first entry contains the mean over all Januaries from 2004 to 2017, the second over all Februaries, etc. The [CF conventions](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.10/cf-conventions.html#climatological-statistics) say that this is the correct format to describe climatological time coordinates :smile: